policyengine_canada/parameters/gov/provinces/mb/tax/income/credits/renters_tax_credit/age_amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Manitoba provides different tax credits according to different age groups.
+values:
+  2022-01-01: 65
+metadata:
+  unit: year
+  label: Manitoba tax credit diference with people with different ages
+  reference:
+    - title: Manitoba Information Guide Manitoba Credits Form MB479
+      href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5007-tc/5007-tc-22e.pdf # page=3
```

Comment:
```suggestion
      href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5007-tc/5007-tc-22e.pdf#page=3
```

Reply:
Hi, I marked some parts of reference format may need to be fixed. The sources are from these two documents:
https://docs.google.com/document/d/1BiajMUJZFzG24Ju5lTybCAW8tp59B3E_DNhS8eJkuR0/edit?pli=1

https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/parameters/gov/states/or/tax/income/subtractions/federal_tax_liability/cap/joint.yaml
Hope this would be helpful.